app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    temp = command.split(" ")
+    old_file = temp[1]
+    new_file = temp[2]
+    if old_file == new_file:
+        exit()
```

Comment:
1. no need to use space as arg for split method, it use space by default
2. u can unpack command and files using only one line
3. no need to use exit method here, u can just make a backward condition
4. u should check is "cp" command passed
![image](https://user-images.githubusercontent.com/103201010/220109257-eccb60da-aa27-43c6-8494-19f311849346.png)

Reply:
Could you please clarify your point about unpacking commands? 
I think, using old_file and new_file for variables makes code more descriptive. 
And split method returns a list, so I decided to store it in variable temp for efficiency reasons, just to not split string each time we call the method